libraries/aws_iam_role.rb

```diff
@@ -48,6 +48,18 @@ def inline_policies
     end
   end
 
+  def tags
+    tag_list = []
+    catch_aws_errors do
+      begin
+        tag_list = @aws.iam_client.get_role(role_name: opts[:role_name]).role.tags
+      rescue
```

Comment:
there is no point having both `catch_aws_errors` and `begin; rescue`!
could you remove `begin; rescue` block from here ?

Reply:
@sathish-progress its updated